codegen/generator/go/templates/formatTypeFunc.go

```diff
@@ -0,0 +1,57 @@
+package templates
```

Comment:
For Go files/directory, the idiomatic case is snake_case.
So it would be `format_type_func.go`.
But I would even simplify. The fact that it handles the type formatting, and that it's done with func doesn't add much to the understanding of the package.

I would rename it to `format.go` as it relates to formatting in general.

Reply:
Fixed in https://github.com/dagger/dagger/pull/4415/commits/ce3612c89843a89cca88bc4281b28e821c37869b